src/pages/companies-list/companies-list.js

```diff
@@ -40,7 +40,20 @@ const CompaniesList = () => {
         </Box>
       );
 
-    return <CompaniesListing data={companies} />;
+    return (
+      <>
+        <h2>Empresas</h2>
```

Comment:
Make sure these titles are aligned to the content here and on the other pages as well.